### PR TITLE
Fix CAD templates wrong content

### PR DESCRIPTION
**Pull request by 0xmaayan**
### Checklist

- Do all Lints pass?
  - [ ] Have you ran `pnpm spellcheck`?
  - [ ] Have you ran `pnpm fmt`?
  - [ ] Have you ran `pnpm lint`?

### CODE REVIEW

File: apps/nextra/pages/en/build/create-aptos-dapp/templates/nft-minting-dapp.mdx [hunk 1]
```diff
@@ -6,7 +6,14 @@ import { Callout, FileTree, Steps } from 'nextra/components';
 
 # Create Aptos Dapp NFT Minting Dapp Template
 
-Digital Assets are the NFT standard for Aptos. The NFT minting dapp template provides an end-to-end NFT minting dapp. With this dapp, you can learn how NFTs work on Aptos, how to write a smart contract and how to write a front end connected to the contract. In addition, it offers a beautiful pre-made UI for a NFT minting dapp users can quickly adjust and deploy into a live server.
+[Digital Assets](../../smart-contracts/digital-asset.mdx) are the NFT standard for Aptos. The NFT minting dapp template provides an end-to-end NFT minting dapp. 
+
+With this dapp, you can:
+1. Create a NFT collection on Aptos.
+2. Modify a premade generated front end UI to build a NFT minting dapp.
```
Comment (gregnazario) [inline apps/nextra/pages/en/build/create-aptos-dapp/templates/nft-minting-dapp.mdx:13]:
```suggestion
2. Modify a pre-made UI to build a NFT minting dapp.
```

File: apps/nextra/pages/en/build/create-aptos-dapp/templates/nft-minting-dapp.mdx [hunk 2]
```diff
@@ -6,7 +6,14 @@ import { Callout, FileTree, Steps } from 'nextra/components';
...
+3. Learn how to write a Move contract and front end connected to that contract. 
+
+In addition, it offers a beautiful pre-made UI for a NFT minting dapp users can quickly adjust and deploy into a live server.
```
Comment (gregnazario) [inline apps/nextra/pages/en/build/create-aptos-dapp/templates/nft-minting-dapp.mdx:16]: This is kinda duplicitous of # 2

File: apps/nextra/pages/en/build/create-aptos-dapp/templates/token-minting-dapp.mdx [hunk 3]
```diff
@@ -6,7 +6,14 @@ import { Callout, FileTree, Steps } from 'nextra/components';
 
 # Create Aptos Dapp Token Minting Dapp Template
 
-The Token minting dapp template provides an end-to-end Fungible Asset minting dapp. With this dapp, you can learn how Fungible Assets work on Aptos, how to write a smart contract and how to write a front end connected to the contract. In addition, it offers a beautiful pre-made UI for a Fungible Asset minting dapp users can quickly adjust and deploy into a live server.
+The Token minting dapp template provides an end-to-end [Fungible Asset](../../smart-contracts/digital-asset.mdx) minting dapp. 
+
+With this dapp, you can:
+1. Create a Fungible Asset on Aptos.
+2. Modify a premade generated front end UI to build a Token minting dapp.
```
Comment (gregnazario) [inline apps/nextra/pages/en/build/create-aptos-dapp/templates/token-minting-dapp.mdx:13]: Similar to above

File: apps/nextra/pages/en/build/create-aptos-dapp/templates/token-staking-dapp.mdx [hunk 4]
```diff
@@ -6,7 +6,12 @@ import { Callout, FileTree, Steps } from 'nextra/components';
 
 # Create Aptos Dapp Token Staking Dapp Template
 
-The Token staking dapp template provides an end-to-end Fungible Asset staking dapp. With this dapp, you can learn how Fungible Assets work on Aptos, how to write a smart contract and how to write a front end connected to the contract. In addition, it offers a beautiful pre-made UI for a Fungible Asset staking dapp users can quickly adjust and deploy into a live server.
+The Token staking dapp template provides an end-to-end [Fungible Asset]((../../smart-contracts/digital-asset.mdx)) staking dapp. 
+
+With this dapp, you can:
+1. Create an incetivize pool on Aptos.
+2. Modify a premade generated front end UI to build a Staking dapp.
```
Comment (gregnazario) [inline apps/nextra/pages/en/build/create-aptos-dapp/templates/token-staking-dapp.mdx:13]: Similar as above

**Review by gregnazario — APPROVED** [review]
There's some verbose / confusing wording, but otherwise it looks good.

**Comment by 0xmaayan** [comment]
@gregnazario addressed comments in https://github.com/aptos-labs/developer-docs/pull/555

**Pull request merged**